test/functional/tests/ci/test_basic.py

```diff
@@ -32,3 +40,40 @@ def test_cas_version():
         if splitted_version[1] != file_cas_version:
             TestRun.LOGGER.error(f"""Version of {splitted_version[0]} from cmd doesn't match
              with file. Expected: {file_cas_version} Actual: {splitted_version[1]}""")
+
+
+@pytest.mark.require_disk("cache_1", DiskTypeSet([DiskType.nand, DiskType.optane]))
```

Comment:
is this test for CI? If yes please add proper pytest mark. If not just answear and I will marge this PR:)

Reply:
If I've checked correctly, looks like this should be part of new CI scpe, so please add:
@pytest.mark.CI